app/slackimport.go

```diff
@@ -798,7 +803,9 @@ func (a *App) OldImportIncomingWebhookPost(post *model.Post, props model.StringI
 				if attachments, success := val.([]*model.SlackAttachment); success {
 					model.ParseSlackAttachment(post, attachments)
 				}
-			} else if key != "from_webhook" {
+				continue
+			}
+			if key != "from_webhook" {
```

Comment:
Same comment as earlier, but I don't think this change is necessary.

Reply:
Agreed (see the comment before)